### PR TITLE
Home 2573 raml

### PR DESCRIPTION
**Pull request by DatChive**
Moving /Account/ to its own directory and created /Accounts/ directory with newly added GET externalFundingAccounts.

image
image
image
image

### CODE REVIEW

**Comment by DatChive** [comment]
Currently the route is `/v4/consumer/accounts/{accountId}/externalFundingAccounts`

`/accounts/` was chosen instead of `/account/` because that's how V3 did it. I'm not married to either and can change if necessary.

**Comment by zrangerz** [comment]
Do you want to have a pedantic verb vs noun REST argument? Cause nerds?

--
Steve Mays
Sent with Airmail

On October 16, 2017 at 11:56:04 AM, Christian Gerrard (notifications@github.com<mailto:notifications@github.com>) wrote:

Currently the route is /v4/consumer/accounts/{accountId}/externalFundingAccounts

/accounts/ was chosen instead of /account/ because that's how V3 did it. I'm not married to either and can change if necessary.

File: api/v4/accounts/examples/externalFundingAccounts.res.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "externalFundingAccounts": [
+    {
+      "id": "218763DGSHGD8reuwasDKJH23",
+      "bankName": "ABC Bank",
+      "bankAccountNumber": "123456789",
```
Comment (kvhswong) [inline api/v4/accounts/examples/externalFundingAccounts.res.json]: Do we need to expose this back to the UI?

File: api/v4/accounts/externalFundingAccounts.raml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  - tokenized
+  - expiry
+
+get:
+  description: |
+      ### Gets a list of external funding accounts
+  queryParameters:
+    custodian:
+      enum:
+        - TD_VEO
```
Comment (kvhswong) [inline api/v4/accounts/externalFundingAccounts.raml]: Are the query parameters necessary since the account will be tied to one custodian?

**Pull request merged**